mytimer/params.py

```diff
@@ -163,6 +163,18 @@
         "second": 0,
         "message": "Time to work (25 mins)",
     },
+    "52_17": {
```

Comment:
"We never used `_` in the program name.
I suggest `52/17` or `52-17`

Reply:
You are right. I was just thinking about potential problems using `-` or `/` would make when users input the program name in CLI.

Reply:
I tested it and using `-` is pretty fine (at least in Linux terminal), please test it on Windows, too.
I will double-check in macOS terminal before merging.